ddp-workspace/projects/ddp-testboston/src/app/components/welcome/welcome.component.html

```diff
@@ -3,11 +3,10 @@
         <div class="content intro-block">
             <h1 class="intro-block__title semi-bold center" translate>HomePage.IntroSection.Title</h1>
             <p class="intro-block__text" translate>HomePage.IntroSection.Text</p>
-            <div *ngIf="!isAuthenticated" class="invitation">
+            <div *ngIf="!isAuthenticated" class="invitation" style="display:flex;justify-content:center;">
```

Comment:
Went with centering after review with Erin

Reply:
Instead of writing inline styles, I'd add styles into `.invitation` class.

Reply:
In this case, I fear that this is going to cascade into needing to do a lot more sass refactoring and testing, so I'd like to keep it (and I admit it's not great).